gui/gui.go

```diff
@@ -37,6 +38,37 @@ type GUI struct {
 	resizeLock        *sync.Mutex
 }
 
+// RecalculateDpiScale calculates dpi scale in comparison with "standard" monitor's dpi values
+func (g *GUI) RecalculateDpiScale() {
+	const standardDpi = 96
+	const mmPerInch = 25.4
+	m := glfw.GetPrimaryMonitor()
```

Comment:
```suggestion
	m := gui.window.GetMonitor()
```

Get the scale of the monitor the window is on, not the primary monitor.

Reply:
@liamg it's turned out that glfw doesn't provide easy to use API to resolve the monitor with currently opened window (`Window.GetMonitor()` is available, but it works only when window is in fullscreen mode). Current implementation calculates overlapping between application window and connected monitors bounds in composed virtual screen coordinates. The most overlapped monitor by app's window should be determined as currently in use by the app. I was not able to test the code much enough because of hardware issues.

Reply:
Ah, that's annoying, I like your solution though.